docs/developer-docs/latest/developer-resources/database-apis-reference/graphql-api.md

```diff
@@ -0,0 +1,471 @@
+
+Single entries can be found by their `id`.
+
+:::request Example query: Find the entry with id 1, only the French fields
+```graphql
+query {
+  document(id: 1, locale: "fr", publicationState: LIVE) {
+    data {
+      id
+      attributes {
+        title
+        locale
+        categories {
+          data {
+            id
+            attributes {
+              name
+            }
+          }
+          meta {
+            pagination {
+              page
+              pageSize
+              total
+              pageCount
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+```
+
+:::
+
+### Fetch multiple entries
+
+:::request Example query: Find all documents and populate 'categories' relation with the `name` attribute
+
+```graphql
+query {
+  documents {
+    data {
+      id
+      attributes {
+        title
+        locale
+        categories {
+          data {
+            id
+            attributes {
+                name
+            }
+          }
+        }
+      }
+    }
+    meta {
+      pagination {
+        page
+        pageSize
+        total
+        pageCount
+      }
+    }
+  }
+}
+
+```
+
+:::
+
+<!-- ? is this part 👇 still relevant? -->
+<!-- ### Fetch dynamic zone data
+
+Dynamic zones are union types in graphql so you need to use fragments to query the fields.
+
+:::request Query
+```graphql
+query {
+  restaurants {
+    dz {
+      __typename
+      ... on ComponentDefaultClosingperiod {
+        label
+      }
+    }
+  }
+}
+```
+::: -->
+
+## Mutations
+
+| Argument     | Type   | Description      |
+| ------------ | ------ | ---------------- |
+| `id`         | String | Entry id         |
+| `attributes` | Object | Entry attributes |
+
+### Create a new entry
+
+:::request Mutation
+
+```graphql
+mutation createArticle {
+  createArticle(data: { title: "Hello", relation: 1 }) {
+    data {
+      id
+      attributes {
+        title
+      }
+    }
+  }
+}
+```
+
+:::
+
+<!-- ? is it still relevant 👇 ? -->
+<!-- The implementation of the mutations also supports relational attributes. For example, you can create a new `User` and attach many `Restaurant` to it by writing your query like this:
+
+:::request Mutation
+
+```graphql
+mutation {
+  createUser(
+    input: {
+      data: {
+        username: "John"
+        email: "john@doe.com"
+        restaurants: ["5b51e3949db573a586ad22de", "5b5b26619b0820c1c2fb79c9"]
+      }
+    }
+  ) {
+    user {
+      username
+      email
+      restaurant {
+        name
+        description
+        price
+      }
+    }
+  }
+}
+```
+
+::: -->
+
+### Update an existing entry
+
+:::request Mutation
+
+```graphql
+mutation updateArticle {
+  updateArticle(id: "1", data: { title: "Hello", relation: 1 }) {
+    data {
+      id
+      attributes {
+        title
+      }
+    }
+  }
+}
+```
+
+:::
+
+<!-- ? is it still relevant? -->
+<!-- You can also update relational attributes by passing an ID or an array of IDs (depending on the relationship).
```

Comment:
Yes but all the examples shouldn't use ObjectIDs as those are MongoDB. Our examples should use integer-based IDs as that is what the SQL databases use

Reply:
+1 for Mongo's ObjectIDs, also we're using the GraphQL's ID scalar which can either be a String or a Number (either way, it'll be casted as a String at the end)